client/.env.docker

```diff
@@ -10,4 +10,4 @@ GITHUB_TOKEN=215dc8f59e98caaa8a3ee2674736b6ca0aad14ca
 
 #pagai
 PYROG_URL=http://pyrog-server:4000
-PYROG_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJjazV4bmp0MGwwMDAwZ2F1dHI0YjYzdmpyIiwiaWF0IjoxNTgwMjAyOTI5fQ.5OJ9WKmlT4jhNqSlDP9ppbST1R6WR_Mnn0SXAsWPjXw
\ No newline at end of file
+PYROG_TOKEN=<token>
```

Comment:
What about the github token for cleaning-scripts?

Reply:
right